buildpacks/nodejs-npm-install/src/main.rs

```diff
@@ -90,6 +90,8 @@ impl Buildpack for NpmInstallBuildpack {
         let build_result = configure_default_processes(&package_json, section.as_ref());
         let logger = section.end_section();
 
+        context.handle_layer(layer_name!("npm_runtime_config"), NpmRuntimeConfigLayer {})?;
```

Comment:
Should the config be set up in the engine buildpack instead, given that's where npm is installed/made ready?

Reply:
I meant `nodejs-npm-engine`.

Ah right. `ETOOMANYBUILDPACKS` :-(